code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm

```diff
@@ -1306,6 +1306,69 @@
 		M.make_dizzy(5)
 		M.make_jittery(5)
 
+/decl/reagent/sanasomnum
+	name = "Sanasomnum"
+	description = "Not strictly a drug, Sanasomnum is actually a cocktail of biomechanical stem cells, which induce a regenerative state of unconsciousness capable healing almost any injury in minutes - however, usage nearly guarantees long-term and irreversible complications, and it is banned from medical use throughout the spur."
+	reagent_state = SOLID
+	color = "#b2db5e"
+	overdose = REAGENTS_OVERDOSE
+	scannable = TRUE
+	taste_description = "blood"
+
+/decl/reagent/sanasomnum/affect_chem_effect(var/mob/living/carbon/M, var/alien, var/removed, var/datum/reagents/holder)
+	. = ..()
+	if(.)
+		M.add_chemical_effect(CE_ORGANREPAIR, 15)
+		M.add_chemical_effect(CE_BLOODRESTORE, 15)
+		M.add_chemical_effect(CE_BLOODCLOT, 15)
+		M.add_chemical_effect(CE_BRAIN_REGEN, 15)
+		M.add_chemical_effect(CE_OXYGENATED, 15)
+		M.add_chemical_effect(CE_ANTITOXIN, 15)
+		M.add_chemical_effect(CE_ANTIBIOTIC, 15)
+		M.add_chemical_effect(CE_SEDATE, 30)
+		M.add_chemical_effect(CE_STABLE, 15)
+		M.add_chemical_effect(CE_UNDEXTROUS, 30)
+		M.add_chemical_effect(CE_CLUMSY, 30)
+
+/decl/reagent/sanasomnum/affect_blood(var/mob/living/carbon/M, var/alien, var/removed, var/datum/reagents/holder)
+	M.adjustCloneLoss(-20 * removed)
+	M.adjustOxyLoss(-2 * removed)
+	M.heal_organ_damage(20 * removed, 20 * removed)
+	M.adjustToxLoss(-1 * removed)
+	if(M.is_asystole() && prob(20))
+		M.resuscitate()
+	if(M.chem_doses[type] > 3)
+		M.status_flags &= ~DISFIGURED
+	if(M.chem_doses[type] > 10)
+		M.make_dizzy(5)
+		M.make_jittery(5)
+	var/mob/living/carbon/human/H = M
+	if(istype(H) && (H.species.flags & NO_BLOOD))
+		return
+	var/dose = M.chem_doses[type]
+	if(dose == metabolism)
+		M.confused += 2
+		M.drowsiness += 2
+	else if(dose < 2)
+		M.Weaken(30)
+		M.eye_blurry = max(M.eye_blurry, 10)
+	else
+		M.sleeping = max(M.sleeping, 30)
```

Comment:
Duplicate effect - you already have CE_SEDATE given as a chem effect. Pick one of the two.